### PR TITLE
feat(Blockchain): add VMware Blockchain integration

### PR DESCRIPTION
**Pull request by svinothini24**

### CODE REVIEW

**Comment by svinothini24** [comment]
@gangadharaswamy @srinivas-kandula @Margarita-Staneva Please review

**Review by gangadharaswamy — APPROVED** [review]

File: vmware_blockchain/README.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+## VMware Blockchain Setup
+
+VMware Blockchain is an enterprise-grade blockchain platform that enables multi-party workflows. It uses advanced techniques, such as byzantine fault-tolerant state machine replication, authenticated data structures, and integration with smart contract execution engines, to enable customers to build and run decentralized multi-party applications.
+
+### Step 1. Install the Wavefront Proxy
+
+If you do not have a [Wavefront proxy](https://docs.wavefront.com/proxies.html) installed on your network, install a proxy. Navigate to Browse > Proxies, click Add new proxy, and follow the instructions.
```
Comment (Margarita-Staneva) [inline vmware_blockchain/README.md:7]: **Browse > Proxies** and **Add new proxy** should be bold, because they are UI elements.

**Review by Margarita-Staneva — COMMENTED** [review]
1 minor comment. Everything else looks good to me

**Comment by gangadharaswamy** [comment]
Rajmohan has created a new PR. Hence closing this PR.

**Pull request closed without merging**